### PR TITLE
Toolchain: Update gcc to 9.3.0

### PR DESCRIPTION
**Pull request by shannonbooth**
Ever closer to C++20! Also fix up some of those pesky "'s

### CODE REVIEW

**Comment by awesomekling** [comment]
Please also update Ports/gcc at the same time :)

**Comment by shannonbooth** [comment]
@awesomekling ah thanks, forgot about that!

File: Toolchain/BuildIt.sh [hunk 1]
```diff
@@ -176,11 +176,11 @@ pushd "$DIR/Build/"
     popd
 
     pushd gcc
-        if [ `uname -s` = "OpenBSD" ]; then
-            perl -pi -e 's/-no-pie/-nopie/g' "$DIR"/Tarballs/gcc-9.2.0/gcc/configure
+        if [ "$(uname -s)" = "OpenBSD" ]; then
+            perl -pi -e 's/-no-pie/-nopie/g' "$DIR/Tarballs/gcc-$VERSION/gcc/configure"
```
Comment (TomMalbran) [inline Toolchain/BuildIt.sh]: Should this be $GCC_VERSION?

Comment (shannonbooth) [reply Toolchain/BuildIt.sh]: Thanks! Big brain fart D:

**Pull request merged**